src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,12 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
```

Comment:
Not fixed
![2023-08-31_15h20_18](https://github.com/mate-academy/react_static-list-of-posts-js/assets/61904995/8ce6b78e-e76a-4068-8d3a-7b9cf368bbbc)

```suggestion
export const CommentInfo = ({ comment: { name, email, body } }) => (
  <div className="CommentInfo">
```